applications/StructuralMechanicsApplication/custom_elements/membrane_element.cpp

```diff
@@ -1008,38 +1009,104 @@ void MembraneElement::Calculate(const Variable<Matrix>& rVariable, Matrix& rOutp
     }
 }
 
-void MembraneElement::CalculateMassMatrix(MatrixType& rMassMatrix,
+void MembraneElement::CalculateConsistentMassMatrix(MatrixType& rMassMatrix,
     const ProcessInfo& rCurrentProcessInfo)
 {
-    KRATOS_TRY
-    auto& r_geom = GetGeometry();
+    KRATOS_TRY;
+    const auto& r_geom = GetGeometry();
+    const SizeType number_of_nodes = r_geom.size();
+    const SizeType dimension = r_geom.WorkingSpaceDimension();
+    const SizeType number_dofs = dimension*number_of_nodes;
+    rMassMatrix = ZeroMatrix(number_dofs,number_dofs);
```

Comment:
missing resize if necessary
also missing `noalias`

Reply:
resize is done in CalculateMassMatrix

Reply:
I'll just delete the whole line  `rMassMatrix = ZeroMatrix(number_dofs,number_dofs);` as this is also done in CalcualteMassMatrix